src/measure_theory/measure/lebesgue.lean

```diff
@@ -630,3 +630,26 @@ begin
 end
 
 end summable_norm_Icc
+
+/-! ### Substituting `-x` for `x`
+
+These lemmas are stated in terms of either `Iic` or `Ioi` (neglecting `Iio` and `Ici`) to match
+mathlib's conventions for integrals over finite intervals (see `interval_integral`). -/
```

Comment:
```suggestion
/-!
### Substituting `-x` for `x`

These lemmas are stated in terms of either `Iic` or `Ioi` (neglecting `Iio` and `Ici`) to match
mathlib's conventions for integrals over finite intervals (see `interval_integral`). For the case
of interval integrals, see `intergral_interval.integral_comp_neg`.
-/

```
Just to make it clear that this is not a docstring and therefore not directly attached to the lemma below

Reply:
It looks like you dropped the `interval_integral.integral_comp_neg` part; having cross-references in doc-strings is helpful, as readers of the web docs can click to jump to the other lemmas.